### ISSUE TITLE
sep: add instructions about what happens if a closing or declaration tx is not signed by other party

### ISSUE THREAD

**Issue by leighmcculloch** [stellar/experimental-payment-channels#27]
The SEP does not discuss what the parties do if one participant proposes and signs a closing transaction and the other participant does not.

What needs to happen in that case is for the iteration number to be skipped. This is implicitly what the SEP requires because whenever `i` is incremented, it is never reverted anywhere, but this should be made more explicit.

The reason not reusing an iteration is important is because one participant has already signed the closing transaction, and so the other participant could resign it at anytime and if the iteration number is reused there would be a valid declaration transaction for that old closing transaction.

**Comment by leighmcculloch** [comment]
This is no longer relevant after #176.
